<ui version="4.0" >
 <class>QAudioServerTestBase</class>
 <widget class="QWidget" name="QAudioServerTestBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QAudioServerTest</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <widget class="QLabel" name="Title" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>220</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text" >
      <string>Audio Management Test</string>
     </property>
     <property name="scaledContents" >
      <bool>false</bool>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QPushButton" name="Audio1" >
       <property name="text" >
        <string>Audio1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QProgressBar" name="progressBar1" >
       <property name="value" >
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QRadioButton" name="AudioMedia1" >
       <property name="text" >
        <string>Media</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
       <property name="autoExclusive" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item rowspan="3" row="1" column="1" >
      <widget class="QTextEdit" name="Status1" />
     </item>
     <item row="2" column="0" >
      <widget class="QRadioButton" name="AudioRingTone1" >
       <property name="text" >
        <string>RingTone</string>
       </property>
       <property name="autoExclusive" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QRadioButton" name="AudioPhone1" >
       <property name="text" >
        <string>Phone</string>
       </property>
       <property name="autoExclusive" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0" >
      <widget class="QPushButton" name="Audio2" >
       <property name="text" >
        <string>Audio2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QProgressBar" name="progressBar2" >
       <property name="value" >
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0" >
      <widget class="QRadioButton" name="AudioMedia2" >
       <property name="text" >
        <string>Media</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
       <property name="autoExclusive" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item rowspan="3" row="5" column="1" >
      <widget class="QTextEdit" name="Status2" />
     </item>
     <item row="6" column="0" >
      <widget class="QRadioButton" name="AudioRingTone2" >
       <property name="text" >
        <string>RingTone</string>
       </property>
       <property name="autoExclusive" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0" >
      <widget class="QRadioButton" name="AudioPhone2" >
       <property name="text" >
        <string>Phone</string>
       </property>
       <property name="autoExclusive" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
